apps/wallet-mobile/src/legacy/Catalyst/DownloadCatalyst.tsx

```diff
@@ -90,23 +95,26 @@ export const DownloadCatalyst = ({onNext}: Props) => {
       <Actions>
         <Button onPress={onNext} title={strings.continueButton} disabled={fund.status.registration !== 'running'} />
       </Actions>
-
-      <StandardModal
-        visible={showModal}
-        title={strings.attention}
-        onRequestClose={() => setShowModal(false)}
-        primaryButton={{
-          label: strings.iUnderstandButton,
-          onPress: () => setShowModal(false),
-        }}
-        showCloseIcon
-      >
-        <Text>{strings.stakingKeyNotRegistered}</Text>
-      </StandardModal>
     </SafeAreaView>
   )
 }
 
+const WarningModal = () => {
+  const strings = useStrings()
+  const styles = useStyles()
+  const {closeModal} = useModal()
+
+  return (
+    <View>
```

Comment:
if you update with develop you need to add horizontal margin

`...atoms.px_lg`

Reply:
Just did to this and the insuficient funds one